provider/server/server_suite_test.go

```diff
@@ -28,13 +28,14 @@ import (
 )
 
 const (
-	eventuallyTimeout    = 30 * time.Second
-	pollingInterval      = 50 * time.Millisecond
-	consistentlyDuration = 1 * time.Second
-	machineClassx3xlarge = "x3-xlarge"
-	machineClassx2medium = "x2-medium"
-	baseURL              = "http://localhost:8080"
-	streamingAddress     = "127.0.0.1:20251"
+	eventuallyTimeout       = 30 * time.Second
+	pollingInterval         = 50 * time.Millisecond
+	gracefulShutdownTimeout = 20 * time.Second
```

Comment:
it is too small for graceful shutdown

Reply:
We have given `eventuallyTimeout` as 30s, which seems to be sufficient. So we have to give the `gracefulShutdownTimeout` anything less than 30s here.

Reply:
it depends on os Image, i recommend create new app run for deletion tests.